src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -109,14 +125,6 @@ class LedgerRPC_test : public beast::unit_test::suite
         env.trust(USD(1000), bob);
         env.close();
 
-        {
```

Comment:
I'd leave this here and change it to something like a garbage string, like "potato". I'd also consider adding a test for a negative ledger index (you can pass it as either a string or an integer)